cmd/cli/commands/deals.go

```diff
@@ -15,17 +15,17 @@ var (
 
 func init() {
 	dealsListCmd.PersistentFlags().Uint64Var(&dealsSearchCount, "limit", 10, "Deals count to show")
-	dealsFinishCmd.PersistentFlags().BoolVar(&addToBlacklist, "blacklist", false, "Add counterparty to blacklist")
+	dealsRemoveCmd.PersistentFlags().BoolVar(&addToBlacklist, "blacklist", false, "Add counterparty to blacklist")
 
-	nodeDealsRootCmd.AddCommand(
+	dealsRootCmd.AddCommand(
 		dealsListCmd,
 		dealsStatusCmd,
-		dealsOpenCmd,
-		dealsFinishCmd,
+		dealsCreateCmd,
```

Comment:
"create" also sounds inaccurate. i'd stick to organizing fist level commands (e.g.  market -> orders, introducing tokens first level command etc). "open" and "finish" (or "close") look perfectly well for "deals" first level command